build.sbt

```diff
@@ -41,30 +41,31 @@ lazy val microservice = (project in file("."))
 
 lazy val appDependencies: Seq[ModuleID] = compile ++ test
 
-lazy val hmrcBootstrapPlay26Version = "1.4.0"
-lazy val hmrcTestVersion = "3.9.0-play-26"
+lazy val hmrcBootstrapPlay28Version = "5.4.0"
 lazy val scalaJVersion = "2.4.1"
-lazy val scalatestPlusPlayVersion = "3.1.2"
+lazy val scalatestPlusPlayVersion = "4.0.3"
 lazy val mockitoVersion = "1.10.19"
 lazy val wireMockVersion = "2.21.0"
 
 
 lazy val compile = Seq(
   ws,
-  "uk.gov.hmrc" %% "bootstrap-play-26" % hmrcBootstrapPlay26Version
+  "uk.gov.hmrc" %% "bootstrap-backend-play-28" % hmrcBootstrapPlay28Version
 )
 
 lazy val test = Seq(
+  "uk.gov.hmrc" %% "bootstrap-test-play-28" % "5.4.0" % Test,
   "org.pegdown" % "pegdown" % "1.6.0" % "test",
-  "org.webjars" %% "webjars-play" % "2.6.3" % "test", 
-  "uk.gov.hmrc" %% "hmrctest" % hmrcTestVersion % "test",
+  "org.webjars" %% "webjars-play" % "2.8.8" % "test",
+  "org.scalatest" %% "scalatest" % "3.2.9" % Test,
+  "com.typesafe.play" %% "play-test" % PlayVersion.current % "test",
+  "org.mockito" %% "mockito-scala-scalatest" % "1.16.46" % Test,
   "org.scalaj" %% "scalaj-http" % scalaJVersion % "test",
   "org.scalatestplus.play" %% "scalatestplus-play" % scalatestPlusPlayVersion % "test",
-  "org.mockito" % "mockito-core" % mockitoVersion % "test",
-  "com.typesafe.play" %% "play-test" % PlayVersion.current % "test",
+  "com.vladsch.flexmark" % "flexmark-all" % "0.35.10" % "test",
```

Comment:
bootstrap-test-play includes stuff like scalatest and flexmark as transitive dependencies so you shouldn't need them.

https://github.com/hmrc/bootstrap-play/blob/main/project/LibDependencies.scala#L70